.evg.yml

```diff
@@ -531,17 +531,26 @@ functions:
         bucket: translators-connectors-releases
 
   "build jdbc driver":
-    command: shell.exec
-    type: test
-    params:
-      shell: bash
-      working_dir: mongo-jdbc-driver
-      script: |
-        ${PREPARE_SHELL}
-        if [[ "${triggered_by_git_tag}" != "" ]]; then
-          EXTRA_PROP="-PisTagTriggered"
-        fi
-        ./gradlew -Dorg.gradle.java.home=${JAVA_HOME} $EXTRA_PROP clean -x test -x integrationTest spotlessApply build shadowjar --rerun-tasks
+    - command: shell.exec
+      type: test
+      params:
+        shell: bash
+        working_dir: mongo-jdbc-driver
+        script: |
+          #./gradlew -Dorg.gradle.java.home=${JAVA_HOME} generateLicenseReport
```

Comment:
Can you swap the 2 lines to make sure that we always have JAVA_HOME defined and controlled.
And remove the dead code.